data-es5.js

```diff
@@ -67,8 +67,7 @@ exports.tests = [
       webkit: true,
       chrome5: null,
       chrome7: true,
-      opera10_50: null,
-      opera12_10: true,
```

Comment:
Let's keep things explicit where possible (keep `opera12_10: true`) here and elsewhere